### PR TITLE
Locations view UI tweaks

### PR DESCRIPTION
**Pull request by EasyRhinoMSFT**

### CODE REVIEW

File: src/Sarif.Viewer.VisualStudio.Core/Views/Locations.xaml [hunk 1]
```diff
@@ -184,9 +184,14 @@
                                                                     <Setter Property="Foreground"
                                                                             Value="{DynamicResource {x:Static vs_shell:CommonDocumentColors.ListItemTextBrushKey}}"/>
                                                                 </DataTrigger>
+                                                                <DataTrigger Binding="{Binding IsSelectedNode}"
```
Comment (yongyan-gh) [inline src/Sarif.Viewer.VisualStudio.Core/Views/Locations.xaml:187]: >IsSelectedNode [](http://example.com/codeflow?start=95&length=14)

whats the difference between `IsSelected` and `IsSelectedNode` here?

Comment (EasyRhinoMSFT) [reply src/Sarif.Viewer.VisualStudio.Core/Views/Locations.xaml:187]: IsSelected indicates the location was navigated to. IsSelectedNode means the TreeViewItem is selected.

**Review by yongyan-gh — COMMENTED** [review]
When the treeview node is selected and mouse cursor over it. the location string is not easy to read

image

image

image

**Comment by EasyRhinoMSFT** [comment]
Ah yes, that only happens in the preview build, forgot to investigate. I'll try to fix.

---
In reply to: [1102672192](https://github.com/microsoft/sarif-visualstudio-extension/pull/538#issuecomment-1102672192) [](http://example.com/codeflow?ancestors=1102672192,1102672192)

**Comment by yongyan-gh** [comment]
got its not our issue then

---
In reply to: [1242281369](https://github.com/microsoft/sarif-visualstudio-extension/pull/538#issuecomment-1242281369) [](http://example.com/codeflow?ancestors=1242281369,1102672192)

**Review by yongyan-gh — APPROVED** [review]

**Pull request merged**